tools/srcServer.js

```diff
@@ -0,0 +1,30 @@
+import express from 'express';
+import path from 'path';
+import webpack from 'webpack';
+import config from '../wepback.config.dev';
+import open from 'open';
+
+/* eslint-disable no-console */
+
+const port = process.env.PORT || 3000;
+const app = express();
+const compiler =webpack(config);
+
+app.use(require("webpack-dev-middleware")(compiler, {
+    noInfo: true,
+    publicPath: config.output.publicPath
+}));
+
+app.use(require("webpack-hot-middleware")(compiler));
+
+app.get('*', function(req, res){
+  res.sendFile(path.join(__dirname, '../src/index.html'));
+});
+
+app.listen(port, function(err){
+  if (err){
+    console.log(err);
```

Comment:
Are we using console statement ? Can we use `logger` instead for development purpose?